python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -67,3 +67,65 @@ def __post_init__(self):
             assert all(
                 isinstance(item, torch.Tensor) for item in self.node_pairs
             ), "Nodes in pairs should be of type torch.Tensor."
+
+
+@dataclass
+class SampledSubgraphImpl(SampledSubgraph):
+    r"""Sampled subgraph of CSCSamplingGraph.
+
+    Examples
+    --------
+    >>> node_pairs = {"A:relation:B": CSCFormatBase(indptr=torch.tensor([0, 1, 2, 3]),
```

Comment:
make sure all the node_pairs related to CSCFormatBase are renamed to csc_format.